addons/binding/org.openhab.binding.vallox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,415 @@
+			<channel id="FirePlaceBoosterClosed" typeId="FirePlaceBoosterClosed" />
+			<channel id="IncommingCurrent" typeId="IncommingCurrent" />
+			<channel id="LastErrorNumber" typeId="LastErrorNumber" />
+		</channels>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<description>
+				    Host name or IP address of RS485-to-Ethernet gateway. 
+				    The gateway needs to be configured as TCP server and the serial 
+				    connection to the vallox with 9600 baud, 8-N-1.
+                </description>
+				<default>lwip</default>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<label>Port</label>
+				<description>TCP Port that the RS485-to-Ethernet Gateway listens to.</description>
+				<default>26</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="FanSpeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed</label>
+		<description>Speed of the fan, an integer value of 1 to 8</description>
+		<category>Fan</category>
+		<state min="1" max="8" step="1" pattern="%d" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="FanSpeedMax">
+		<item-type>Number</item-type>
+		<label>FanSpeedMax</label>
+		<category>Fan</category>
+		<state min="1" max="8" step="1" pattern="%d" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="FanSpeedMin">
+		<item-type>Number</item-type>
+		<label>FanSpeedMin</label>
+		<category>Fan</category>
+		<state min="1" max="8" step="1" pattern="%d" readOnly="false"></state>
+	</channel-type>
+
+	<channel-type id="TempInside">
+		<item-type>Number</item-type>
+		<label>Temp Inside</label>
+		<description>Current temperature inside the building.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="0.1" pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="TempOutside">
+		<item-type>Number</item-type>
+		<label>Temp Outside</label>
+		<description>Current temperature outside the building.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="0.1" pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="TempExhaust">
+		<item-type>Number</item-type>
+		<label>Temp Exhaust</label>
+		<description>Current temperature of the air flow exhausting the building.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="0.1" pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="TempIncomming">
+		<item-type>Number</item-type>
+		<label>Temp Incomming</label>
+		<description>Current temperature of the air flow incomming to the building.</description>
+		<category>Temperature</category>
+		<state min="-20" max="40" step="0.1" pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="InEfficiency">
+		<item-type>Number</item-type>
+		<label>InEfficiency</label>
```

Comment:
What is this? Anyhow, don't use CamelCase in labels...

Reply:
All these efficiencies are percent calculations from different ventilation system temperatures. Per manufacturer description.